changelog/fragments/liveness_readiness_probe_for_operator.yaml

```diff
@@ -0,0 +1,47 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      For Ansible/Helm-based operators, add Liveness and Readiness probe.
```

Comment:
```suggestion
      For Ansible/Helm-based operators, add simple Liveness and Readiness probe.
```

Reply:
I'd not like to add simple or not because its implementation came from controller-runtime.